libraries/rust/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "standardwebhooks"
-version = "1.3.0"
+version = "1.0.1"
```

Comment:
```suggestion
version = "1.0.0"
```

Reply:
On purpose unfortunately. I accidentally released a 1.0.0 which was wrong and had to yank it. :(
I know we want all the lib version in sync, and we'll do it once CI is setup to do it.